README.md

```diff
@@ -100,23 +101,30 @@ Create a Connection
-| CONNECTION_TIMEOUT  | No       | Total timeout in seconds when connecting to Snowflake. Default to 120 seconds |
-| AUTHENTICATOR       | No       | The method of authentication. Currently support snowflake(default), [native SSO okta](https://docs.snowflake.net/manuals/user-guide/admin-security-fed-auth-use.html#native-sso-okta-only) and [externalbrowser](https://docs.snowflake.net/manuals/user-guide/admin-security-fed-auth-use.html#browser-based-sso).  |
+| Connection Property       | Required | Comment                                                                       |
+|---------------------------|----------|-------------------------------------------------------------------------------|
+| ACCOUNT                   | Yes      | Account should not include region or clound provider information. i.e. account should be XXX instead of XXX.us-east-1.|
+| DB                        | No       |                                                                               |
+| HOST                      | No       | If no value specified, driver will use \<ACCOUNT\>.snowflakecomputing.com. However, if you are not in us-west deployment, or you want to use global url, HOST is required, i.e. XXX.us-east-1.snowflakecomputing.com, or XXX-jkabfvdjisoa778wqfgeruishafeuw89q.global.snowflakecomputing.com|
+| PASSWORD                  | Depends  | Required for snowflake(default) and native sso okta authentication methods. Ignored for all the other authentication types.|
+| ROLE                      | No       |                                                                               |
+| SCHEMA                    | No       |                                                                               |
+| USER                      | Yes      | For native sso okta and externalbrowser, this should be the login name for your idp.     |
+| WAREHOUSE                 | No       |                                                                               |
+| CONNECTION_TIMEOUT        | No       | Total timeout in seconds when connecting to Snowflake. Default to 120 seconds |
+| AUTHENTICATOR             | No       | The method of authentication. Currently supports snowflake(default), [native SSO okta](https://docs.snowflake.net/manuals/user-guide/admin-security-fed-auth-use.html#native-sso-okta-only), [externalbrowser](https://docs.snowflake.net/manuals/user-guide/admin-security-fed-auth-use.html#browser-based-sso), [snowflake_jwt](https://docs.snowflake.com/en/user-guide/key-pair-auth.html) and [oauth](https://docs.snowflake.com/en/user-guide/oauth.html)<br />- When using *snowflake*: USER and PASSWORD are required.<br />- When using *externalbrowser*: USER is required.<br /> - When using *snowflake_jwt*: PRIVATE_KEY_FILE or PRIVATE_KEY are required. <br />- When using *oauth*: TOKEN is required. <br />- When using *native sso okta*: USER and PASSWORD are required. |
```

Comment:
This might be easier to read if the information on each supported method were in one place. e.g. 

```suggestion
| AUTHENTICATOR             | No       | The method of authentication. Currently supports the following values: <br /> - snowflake (default): You must also set USER and PASSWORD. <br /> - [the URL for native SSO through Okta](https://docs.snowflake.com/en/user-guide/admin-security-fed-auth-use.html#native-sso-okta-only): You must also set USER and PASSWORD. <br /> - [externalbrowser](https://docs.snowflake.com/en/user-guide/admin-security-fed-auth-use.html#browser-based-sso): You must also set USER. <br /> - [snowflake_jwt](https://docs.snowflake.com/en/user-guide/key-pair-auth.html): You must also set PRIVATE_KEY_FILE or PRIVATE_KEY. <br / > - [oauth](https://docs.snowflake.com/en/user-guide/oauth.html): You must also set TOKEN.
```